packages/console/src/hooks/use-api.ts

```diff
@@ -150,19 +160,39 @@ export const useStaticApi = ({
               }
             },
           ],
+          afterResponse: [
+            async (request, _options, response) => {
+              if (
+                isCloud &&
+                isDevFeaturesEnabled &&
+                ['POST', 'PUT', 'DELETE'].includes(request.method) &&
+                response.status >= 200 &&
+                response.status < 300
+              ) {
+                void mutateSubscriptionUsage();
+                void mutateSubscriptionQuota();
+                void mutateScopeResourceUsage();
+                void mutateScopeRoleUsage();
```

Comment:
We may need to encapsulate these operations within a method provided by a hook, otherwise, having to write so many lines of function calls in different places could easily lead to omissions.

Reply:
Good call, updated.